pkgs/top-level/all-packages.nix

```diff
@@ -9860,6 +9860,8 @@ with pkgs;
 
   kyverno = callPackage ../applications/networking/cluster/kyverno { };
 
+  kyverno-chainsaw = callPackage ../application/networking/cluster/kyverno-chainsaw { };
+
```

Comment:
When placing the package under the `pkgs/by-name` tree (see above) the changes in `all-packages.nix` are no longer necessary
```suggestion
```